api_tests/tests/dry/multiple_peers.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * @config { "logDir": "dry" }
+ */
+
+import { threeActorTest } from "../../lib/actor_test_new";
+import { createDefaultSwapRequest } from "../../lib/utils";
+import { expect } from "chai";
```

Comment:
Can you please create a follow-up ticket for moving away from `chai` and using the jest matchers?

Reply:
Created in #2139